packages/serverless-offline-sqs/src/index.js

```diff
@@ -56,6 +53,14 @@ class ServerlessOfflineSQS {
       'before:offline:start:end': this.offlineStartEnd.bind(this)
     };
 
+    this.serverlessOfflinePlugin = this.serverless.pluginManager.plugins.find(
```

Comment:
For me, this doesn't find anything even though I do have `serverless-offline-sqs` plugin installed... Are you sure this is the best way to find it?

Reply:
I couldn't find any better way to hook into the serverless-offline plugin. The plugin order matters. Make sure that `- serverless-offline` appears before `- serverless-offline-sqs` in your `serverless.yml`

Reply:
Yup, that was it - after changing the order I can approve that it works on my side. (tested with serverless-offline versions 6.1.0 and 6.1.4)